docs/sources/send-data/alloy/examples/alloy-kafka-logs.md

```diff
@@ -0,0 +1,373 @@
+    ✔ Container loki-fundamentals-loki-1         Started                        
+    ✔ Container loki-fundamentals-alloy-1        Started
+    ✔ Container loki-fundamentals-zookeeper-1    Started
+    ✔ Container loki-fundamentals-kafka-1        Started
+    ```
+
+We will be access two UI interfaces:
+- Alloy at [http://localhost:12345](http://localhost:12345)
+- Grafana at [http://localhost:3000](http://localhost:3000)
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Step 2: Configure Alloy to ingest raw Kafka logs
+
+In this first step, we will configure Alloy to ingest raw Kafka logs. To do this, we will update the `config.alloy` file to include the Kafka logs configuration.
+
+{{< docs/ignore >}}
+**Note: Killercoda has an inbuilt Code editor which can be accessed via the `Editor` tab.**
+{{< /docs/ignore >}}
+
+### Loki Kafka Source component
+
+First, we will configure the Loki Kafka source. `loki.source.kafka` reads messages from Kafka using a consumer group and forwards them to other `loki.*` components.
+
+The component starts a new Kafka consumer group for the given arguments and fans out incoming entries to the list of receivers in forward_to.
+
+Open the `config.alloy` file in the `loki-fundamentals` directory and copy the following configuration:
+```alloy
+loki.source.kafka "raw" {
+  brokers                = ["kafka:9092"]
+  topics                 = ["loki"]
+  forward_to             = [loki.write.http.receiver]
+  relabel_rules          = loki.relabel.kafka.rules
+  version                = "2.0.0"
+  labels                = {service_name = "raw_kafka"}
+}
+```
+
+In this configuration:
+- `brokers`: The Kafka brokers to connect to.
+- `topics`: The Kafka topics to consume. In this case, we are consuming the `loki` topic.
+- `forward_to`: The list of receivers to forward the logs to. In this case, we are forwarding the logs to the `loki.write.http.receiver`.
+- `relabel_rules`: The relabel rules to apply to the incoming logs. This can be used to generate labels from the temporary internal labels that are added by the Kafka source.
+- `version`: The Kafka protocol version to use.
+- `labels`: The labels to add to the incoming logs. In this case, we are adding a `service_name` label with the value `raw_kafka`. This will be used to identify the logs from the raw Kafka source in the Log Explorer App in Grafana.
+
+For more information on the `loki.source.kafka` configuration, see the [Loki Kafka Source documentation](https://grafana.com/docs/alloy/latest/reference/components/loki.source.kafka/).
+
+### Loki Relabel Rules component
+
+Next, we will configure the Loki relabel rules. The `loki.relabel` component rewrites the label set of each log entry passed to its receiver by applying one or more relabeling rules and forwards the results to the list of receivers in the component’s arguments. In our case we are directly calling the rule from the `loki.source.kafka` component.
```

Comment:
it would help to have a sentence describing why you would want to do this. At the bottom of this section you point out that we're adding `__meta_kafka_topic`.  You might just make some mention of wanting to filter all logs based on those which came from kafka, and that label being a convenient way to do that (or somesuch)